src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -1563,7 +1563,7 @@
     short-summary: Extra nodes used to speed upgrade. When specified, it represents the number or percent used, eg. 5 or 33%
   - name: --node-taints
     type: string
-    short-summary: The node taints for the node pool. You can update the existing node taint of a nodepool or create a new node taint for a nodepool. Pass the empty string `""` to remove all taints.
+    short-summary: The node taints for the node pool. You can update the existing node taint of a nodepool or create a new node taint for a nodepool. Pass the empty string `""` (or `'""'` in PowerShell) to remove all taints.
```

Comment:
This is a PowerShell issue (https://github.com/Azure/azure-cli/issues/15529) and many arguments have similar help messages:

https://github.com/Azure/azure-cli/blob/f305565a270341974a0ae60eb81f4274730cb79e/src/azure-cli/azure/cli/command_modules/vm/_params.py#L1075

https://github.com/Azure/azure-cli/blob/3f51113b641484ed2582aafa5f762afa685bc696/src/azure-cli/azure/cli/command_modules/network/_params.py#L654

I am not sure if it is a good idea to mention it every time an empty string is passed to an argument, as this is often overlooked.

Reply:
Good point. For the general issue of PowerShell, we may consider adding it to a centralized document